### ISSUE TITLE
in taskview, add functionality to delete all tasks

### ISSUE THREAD

**Issue by njthomas** [ranger/ranger#2527]
#### Current Behavior

When viewing tasks in taskview, I can delete one task at a type by hitting "dd".
If there are a lot of tasks, it is tedious to delete all of them individualy.

#### Expected Behavior

There should be some way to delete all tasks

#### Context

Some of the log files in /var/log on a system are large, traversing over these files generates a task "Getting preview of /var/log/foo.log" for each one. It is tedious to delete each task.

#### Possible Solutions

I've created a patch to fix this. It adds  the function task_remove_all() to the TaskView class, and sets a keybinding (dD) to call it.

#### Steps to reproduce
1. start ranger
2. go to any directory with multiple log files larger than 100MiB or so
3. traverse over each file
4. run "w" to go into taskview

**Comment by njthomas** [comment]
Created pull-request #2528 to add this functionality.

**Comment by chu4ng** [comment]
As I say in PR, you can destroy the whole queue. That bind works for me:
`tmap dD eval -q fm.loader.destroy()`

**Comment by njthomas** [comment]
@chu4ng Indeed, that does the trick as well. I'll update my patch to do it that way.

**Comment by toonn** [comment]
Hmm, I'm a little torn on this because it's not really in line with the vim paradigm. Can we make this more similar to deletion of files, i.e., accepting a movement `dgg/dG` or using visual selection `vdd`?

**Comment by njthomas** [comment]
I'm in favor of closing this issue out. In light of several years of usage, this is not that big a problem that typing `dd` a few times can't fix. I don't think anyone else is clamoring for this functionality, and now, neither am I.

**Comment by toonn** [comment]
I'll leave it up to you whether to close the issue or not. It's not a high priority to me but I'm also not opposed to the functionality.
